rancher2/structure_pod_security_policy_run_as_user_test.go

```diff
@@ -0,0 +1,76 @@
+package rancher2
+
+import (
+	"reflect"
+	"testing"
+
+	managementClient "github.com/rancher/types/client/management/v3"
+)
+
+var (
+	testPodSecurityPolicyRunAsUserConf           *managementClient.RunAsUserStrategyOptions
+	testPodSecurityPolicyRunAsUserInterface      []interface{}
+	testNilPodSecurityPolicyRunAsUserConf        *managementClient.RunAsUserStrategyOptions
+	testEmptyPodSecurityPolicyRunAsUserInterface []interface{}
+)
+
+func init() {
+	testPodSecurityPolicyRunAsUserConf = &managementClient.RunAsUserStrategyOptions{
+		Rule:   "RunAsAny",
+		Ranges: testPodSecurityPolicyIDRangesConf,
+	}
+	testPodSecurityPolicyRunAsUserInterface = []interface{}{
+		map[string]interface{}{
+			"rule":  "RunAsAny",
+			"range": testPodSecurityPolicyIDRangesInterface,
+		},
+	}
+	testEmptyPodSecurityPolicyRunAsUserInterface = []interface{}{}
+}
+
+func TestFlattenPodSecurityPolicyRunAsUser(t *testing.T) {
+
+	cases := []struct {
+		Input          *managementClient.RunAsUserStrategyOptions
+		ExpectedOutput []interface{}
+	}{
+		{
+			testPodSecurityPolicyRunAsUserConf,
+			testPodSecurityPolicyRunAsUserInterface,
+		},
+		{
+			testNilPodSecurityPolicyRunAsUserConf,
+			testEmptyPodSecurityPolicyRunAsUserInterface,
+		},
+	}
+
+	for _, tc := range cases {
+		t.Logf("Expected: %#v\nGiven:    %#v", tc.Input, tc.ExpectedOutput)
```

Comment:
Could you please remove this log entry??

Reply:
Yes